### DISCUSSION TITLE
[archived] "Access Token" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#12918] [General]
# en/latest/access-token/

https://argo-workflows.readthedocs.io/en/latest/access-token/

**Comment by tianye0301** [comment]
I followed the steps as following 
To use that token with the CLI you need to set `ARGO_SERVER` (see `argo --help`).
Use that token in your API requests, e.g. to list workflows:
```sh
curl https://localhost:2746/api/v1/workflows/argo -H "Authorization: $ARGO_TOKEN"
```

but it returns an error:
```
curl: (60) SSL certificate problem: unable to get local issuer certificate
```

**Reply by agilgur5** [reply]
> ```
> curl: (60) SSL certificate problem: unable to get local issuer certificate
> ```

Your Server is probably set-up with a self-signed certificate. If you installed via the Quick Start, this is [mentioned there](https://argo-workflows.readthedocs.io/en/latest/quick-start/#submit-via-the-ui).

To workaround this for testing purposes, you can use `curl --insecure`. That is equivalent to `argo --insecure-skip-verify` [in the CLI](https://argo-workflows.readthedocs.io/en/latest/cli/argo/#options) in Server mode (i.e. with `ARGO_SERVER` set).

For production workloads, you'll want a properly CA signed certificate, and ensure your client recognizes the CA (if it is not public).
